examples/stateful_example.yaml

```
@@ -0,0 +1,115 @@
+      type: tumbling
+      size_ms: 60000  # 1 minute windows
+    output:
+      type: file
+      path: "./output/temperature_stats.txt"
+      format: json
+    error_output:
+      type: file
+      path: "./errors/temperature_errors.txt"
+    state:
+      operator_id: "temperature-processor"
+      enabled: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)